src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -130,7 +129,6 @@ private boolean canPaymentWorksNewVendorRequestProcessingContinueForVendor(Payme
         if (pmwNewVendorNotNull(stgNewVendorRequestDetailToProcess, errorMessages)
                 && pmwDtosCouldConvertCustomAttributesToPmwJavaClassAttributes(stgNewVendorRequestDetailToProcess)
                 && pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages)
-                && allPmwNewVendorIsoCountriesMapToSingleFipsCountry(stgNewVendorRequestDetailToProcess, errorMessages)
```

Comment:
This method was performing two different types of checks: Those which verify that the required country codes are present, and those which verify that the country codes map to a single FIPS country. Should a failure still happen at this stage if the country data is absent? Or should the PVEN still be created if the country codes are missing?

Reply:
Yes, we want to create a saved PVEN in both situations (can't convert an ISO country to a FIPS country and lack of country provided).  The PMW form requires various countries as specified by Purchasing staff, so the scenario is not likely to present itself.

Interesting find about legacy form processing mode.  Since we are turning on foreign form processing mode this evening, and won't be going back to legacy mode, it should be fine to fix the scenario you mention.  Also, in the near term, we will be removing legacy form only code.

Reply:
Ok, thanks for clarifying. I'm fine with this change as-is, then.